exporter-helper.php

```diff
@@ -45,8 +46,8 @@ function get_campaign_attachments( $post_ids ) {
 		$values         = [];
 		$values[0]      = $wpdb->postmeta;
 		$values         = array_merge( $values, $post_ids );
-		$prepared_sql   = $wpdb->prepare( $sql, $values );     // WPCS: unprepared SQL OK.
-		$results        = $wpdb->get_results( $prepared_sql ); // WPCS: unprepared SQL OK.
+		$prepared_sql   = $wpdb->prepare( $sql, $values ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
```

Comment:
Probably this phpcs:ignore is not needed, as this is not a `query()` call on this line. And I would assume that for the next line it should be able to figure out that `$prepared_sql` has been prepared, so maybe that's also not needed.

Reply:
Yes, That's true. but phpcs still complain on this 2 lines.
```
FILE: ...ublic/wp-content/themes/planet4-master-theme/exporter-helper.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 49 | ERROR | Use placeholders and $wpdb->prepare(); found $sql
    |       | (WordPress.DB.PreparedSQL.NotPrepared)
 50 | ERROR | Use placeholders and $wpdb->prepare(); found
    |       | $prepared_sql
    |       | (WordPress.DB.PreparedSQL.NotPrepared)
----------------------------------------------------------------------
```